src/core/u-compress.c

```diff
@@ -355,9 +406,10 @@ REBSER *Decompress(
 
     // Since the initialization succeeded, go ahead and make the output buffer
     //
-    REBSER *output = Make_Binary(buf_size);
+    REBSER *output = Make_Binary(sizeof(REBSER*) + buf_size);
+    *cast(REBSER**, BIN_HEAD(output)) = output; // self-referential prefix
     strm.avail_out = buf_size;
-    strm.next_out = BIN_HEAD(output);
+    strm.next_out = BIN_HEAD(output) + sizeof(REBSER*);
 
```

Comment:
I just realized that SER_HEAD didn't count for `bias` (spent a little bit more time on this, and found that it couldn't because of the REBSER might not have dynamic content, thus no `bias` at all). If it did, you could get rid of ` + sizeof(REBSER*)` by setting the bias.